### DISCUSSION TITLE
Best way to configure data bridge

### DISCUSSION THREAD

**Discussion by vibhubithar** [emqx/emqx#9809] [Q&A]
I have an application that has its broker and sends out data on multiple topics. 

I want to check data from specific topics and if needed send it to REST API using webhooks

Ques: should i create data bridge for top most topic and parse it in the SQL 

OR

create data bridge for each topic ?

**Comment by lafirest** [comment]
@vibhubithar 
No, you don't need to do this, you could just `From a topic list`, like this:
`SELECT
  *
FROM
  "t/#", "t2/test", "foo/boo"`

**Reply by vibhubithar** [reply]
So when creating data bridge use top most topic

then use rules to extract data from individual topic

right?

**Reply by lafirest** [reply]
I am not sure what is the meaning of `top most topic`, but actually most of the time we use data bridge with rule engine together

**Reply by vibhubithar** [reply]
let's say my external broker publishes messages on

A/B/C/1/1
A/B/C/1/2
A/B/C/2/1
A/B/C/2/1
A/B/C/3/1
A/B/C/3/2

in EMQX on the data bridge i subscribe to A/B/C/# and publish those messages locally to A/B/C/#

then in the associated rule, i should be able to select messages by sub-topics like 

SELECT
  *
FROM
  "A/B/C/1/#"

But this is not working
